src/mon/LogMonitor.cc

```diff
@@ -403,13 +403,42 @@ bool LogMonitor::preprocess_command(MonOpRequestRef op)
     if (f) {
       f->open_array_section("tail");
     }
+
+    std::string level_str;
+    clog_type level;
+    if (cmd_getval(g_ceph_context, cmdmap, "level", level_str)) {
+      if (level_str == "debug") {
```

Comment:
https://github.com/ceph/ceph/blob/master/src/librados/RadosClient.cc#L946

maybe a helper in common/LogEntry.h.. one that will take 'err' or 'error', etc?